src/style.css

```
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: "Avenir", sans-serif;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir Bold";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-weight: 300;
+}
+
+.form-field {
+  box-sizing: border-box;
+  position: relative;
+  display: flex;
+  align-items: center;
+}
+
+.first-form {
+  width: 100%;
+  margin-top: 19px;
+  margin-bottom: 19px;
+}
+
+.second-form {
+  width: 50%;
+}
+
+.input {
+  font-family: "Avenir", sans-serif;
+  box-sizing: border-box;
+  width: 100%;
+  padding: 25px 26px 23px 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input-first {
```

```suggestion
.input-small {
```
first и second плохие названия классов, потому что положение элементов в html может меняться(а размер - маловероятно)